core/src/main/java/org/wso2/msf4j/ServiceMethodInfo.java

```diff
@@ -18,22 +18,21 @@
 
 
```

Comment:
The interceptors (deprecated) will work as before without any difference. However, internally they function as global request and global response interceptors. To achieve this task ServiceMethodInfo.java was changed appropriately. This class is only used internally (previously and now) when executing interceptors (deprecated)